readthedocs/proxito/views/serve.py

```diff
@@ -57,7 +57,15 @@ def get(self,
             # This is since the final URL will check for authz.
             self.cache_request = True
 
-        return self.system_redirect(request, final_project, lang_slug, version_slug, filename)
+        is_external = getattr(request, "external_domain", False)
```

Comment:
It seems we are just moving the `getattr` ouside the function call. Are these going to be removed in the future and we are going to the these attributes from other place instead of the `request`?

Reply:
yeah, with the new proxito implementation, we won't have those attributes scatter around the request object.